app/src/components/Shared/LinkTo.jsx

```diff
@@ -0,0 +1,31 @@
+import React, { Component } from 'react';
```

Comment:
This class we'll be used in the WHOLE project replacing current arrows for this component. That's the idea.